pom.xml

```diff
@@ -76,17 +76,33 @@
     <modules>
         <module>accesscontroltool-bundle</module>
         <module>accesscontroltool-package</module>
-        <module>accesscontroltool-oakindex-package</module>
-    </modules>    
+    </modules>
 
     <dependencyManagement>
         <dependencies>
+            <!-- 5.6.1 dependencies -->
             <dependency>
-                <groupId>com.adobe.aem</groupId>
-                <artifactId>aem-api</artifactId>
-                <version>6.0.0.1</version>
+                <groupId>org.apache.jackrabbit</groupId>
+                <artifactId>jackrabbit-jcr-commons</artifactId>
+                <version>2.11.2</version>
+            </dependency>
+            <dependency>
+                <groupId>org.apache.sling</groupId>
+                <artifactId>org.apache.sling.settings</artifactId>
+                <version>1.2.2</version>
+            </dependency>
+            <dependency>
+                <groupId>com.day.jcr.vault</groupId>
+                <artifactId>com.day.jcr.vault</artifactId>
+                <version>2.4.32</version>
+            </dependency>
+            <dependency>
+                <groupId>com.day.cq</groupId>
+                <artifactId>cq-commons</artifactId>
+                <version>5.6.4</version>
                 <scope>provided</scope>
             </dependency>
+            <!-- 5.6.1 dependencies end -->
```

Comment:
It was the area replacing the aem 6.0.0 dependency - but I agree it doesn't seem to make a lot of sense now. Let's remove the comment at all.